spark-commons/src/main/scala/za/co/absa/spark/commons/errorhandling/implementations/submits/ErrorMessageSubmitJustErrorValue.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.spark.commons.errorhandling.implementations.submits
+
+import org.apache.spark.sql.functions.{lit, map}
+import org.apache.spark.sql.types.StringType
+import za.co.absa.spark.commons.errorhandling.ErrorMessageSubmit
+import za.co.absa.spark.commons.errorhandling.implementations.submits.ErrorMessageSubmitJustErrorValue.noColumnKey
+import za.co.absa.spark.commons.errorhandling.types._
+
+class ErrorMessageSubmitJustErrorValue(
+                                        val errType: ColumnOrValue[ErrType],
+                                        val errCode: ColumnOrValue[ErrCode],
+                                        val errMsg: ColumnOrValue[ErrMsg],
+                                        errValue: ColumnOrValue[String],
+                                        override val additionInfo: ColumnOrValue[AdditionalInfo] = ColumnOrValue.asEmpty
+                                      ) extends ErrorMessageSubmit {
+  val errColsAndValues: ColumnOrValue[ErrColsAndValues] = ColumnOrValue(map(lit(noColumnKey), errValue.column.cast(StringType)))
+}
+
+object ErrorMessageSubmitJustErrorValue {
+  val noColumnKey: ErrSourceColName = ""
+
+  def apply(errType: ErrType, errCode: ErrCode, errMessage: ErrMsg, errValue: String, additionalInfo: AdditionalInfo = None): ErrorMessageSubmitJustErrorValue = {
```

Comment:
errValue - even this can have ErrValue type, defined in types.scala, and such new type can be part of ErrColsAndValues. I don't insist although it would make all this more consistent

Reply:
it's still a String